js/main.js

```diff
@@ -21,4 +22,84 @@ function save(event) {
   data.entries.unshift(formInput);
   $photoImage.setAttribute('src', 'images/placeholder-image-square.jpg');
   $formJournal.reset();
+  $entryList.appendChild(renderEntry(data.entries[0]));
```

Comment:
Instead of using `data.entries[0]`, you can also just use the `formInput` object you created above.